### PR TITLE
ISAICP-6467: Do not show the Translate button to edit pages

### PR DESCRIPTION
**Pull request by idimopoulos**

### CODE REVIEW

File: config/sync/block.block.webtools_etrans.yml [hunk 1]
```diff
@@ -25,4 +24,9 @@ settings:
   delay: 0
   include: ''
   exclude: ''
-visibility: {  }
+visibility:
+  request_path:
+    id: request_path
+    pages: "/node/*/edit\r\n/node/*/delete\r\n/rdf_entity/*/edit\r\n/rdf_entity/*/delete\r\n/taxonomy_term/*/edit\r\n/taxonomy_term/*/delete"
```
Comment (claudiu-cristea) [inline config/sync/block.block.webtools_etrans.yml:30]: Wouldn't it be more exact to use the route filter?

```yaml
- entity.node.edit_form
- entity.node.delete_form
- ...
```

File: tests/features/machine_translation.feature [hunk 2]
```diff
@@ -32,8 +32,11 @@ Feature: Machine translation
       | state      | validated     |
       | collection | Brexit        |
 
+    When I am logged in as a moderator
     # Inside a collection sidebar, only custom pages, glossary terms (but not
     # the glossary overview) and the about page should be translatable.
+    And I go to the edit form of the "Brexit" collection
+    Then I should not see any Webtools eTrans elements
```
Comment (claudiu-cristea) [inline tests/features/machine_translation.feature:39]: I see we're banning also the entity delete forms, then why aren't we testing that too?

**Review by claudiu-cristea — CHANGES_REQUESTED** [review]
I have one question and a request to expand the tests

File: config/sync/block.block.webtools_etrans.yml [hunk 3]
```diff
@@ -25,8 +25,14 @@ settings:
...
-    pages: "/node/*/edit\r\n/node/*/delete\r\n/rdf_entity/*/edit\r\n/rdf_entity/*/delete\r\n/taxonomy_term/*/edit\r\n/taxonomy_term/*/delete"
+  request_route:
+    id: request_route
+    routes:
+      0: entity.node.delete_form
+      2: entity.node.edit_form
+      4: entity.rdf_entity.delete_form
+      6: entity.rdf_entity.edit_form
+      8: entity.taxonomy_term.edit_form
+      10: entity.taxonomy_term.delete_form
```
Comment (claudiu-cristea) [inline config/sync/block.block.webtools_etrans.yml:36]: This looks like a bug in the route condition module. My guess is that, when exploding the blob, an `array_values()` is missing.

File: tests/src/Traits/EntityTrait.php [hunk 4]
```diff
@@ -75,10 +75,22 @@ protected static function getEntityByLabel(string $entity_type_id, string $label
    */
   protected static function entityTypeAliases(): array {
     return [
+      'asset_distribution' => 'rdf_entity',
+      'asset_release' => 'rdf_entity',
+      'contact_information' => 'rdf_entity',
```
Comment (pfrenssen) [inline tests/src/Traits/EntityTrait.php]: This method should take human readable aliases, so there should not be any underscores here.

**Review by pfrenssen — CHANGES_REQUESTED** [review]

**Review by pfrenssen — APPROVED** [review]

**Pull request merged**